src/main/java/th/ac/kmitl/science/comsci/example/controllers/HelloServlet.java

```diff
@@ -23,6 +23,6 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         out.println("<ul>");
         out.println("<li>Bearchitect</li>");
         out.println("<li>58050237 Chayapol Limanon</li>");
-        out.println("</ul>");
+        out.println("<li>57050309 Mayrika Phaholvech</li>");
```

Comment:
pull the upstream for "57050294 Pichavarit Panyokaew" and "58050276 Thanathat Surakhup" and added the line for "</ul>"